internal/providers/terraform/aws/msk_cluster.go

```diff
@@ -0,0 +1,59 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"strings"
+)
+
+func GetMSKClusterRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_msk_cluster",
+		RFunc: NewMskCluster,
+	}
+}
+
+func NewMskCluster(d *schema.ResourceData, u *schema.ResourceData) *schema.Resource {
+	region := d.Get("region").String()
+
+	brokerNodes := decimal.NewFromInt(d.Get("number_of_broker_nodes").Int())
+	brokerInstance := d.Get("broker_node_group_info.0.instance_type").String()
```

Comment:
```suggestion
	instanceType := d.Get("broker_node_group_info.0.instance_type").String()
```

Reply:
The price component uses "Kafka" so I've added a string replace to uppercase the first letter. I can't use Title as the instance contains "." and produces "Kafka.T3.Small" which fails to retrieve any prices.